promql/functions.go

```diff
@@ -1266,6 +1376,19 @@ func init() {
 		delete(parser.Functions, "xincrease")
 		delete(parser.Functions, "xrate")
 		fmt.Println("Successfully replaced rate & friends with xrate & friends (and removed xrate & friends function keys).")
+	case "2":
+		FunctionCalls["delta"] = FunctionCalls["ydelta"]
+		parser.Functions["delta"] = parser.Functions["ydelta"]
+		parser.Functions["delta"].Name = "delta"
+
+		FunctionCalls["increase"] = FunctionCalls["yincrease"]
+		parser.Functions["increase"] = parser.Functions["yincrease"]
+		parser.Functions["increase"].Name = "increase"
+
+		FunctionCalls["rate"] = FunctionCalls["yrate"]
+		parser.Functions["rate"] = parser.Functions["yrate"]
+		parser.Functions["rate"].Name = "rate"
+		fmt.Println("Successfully replaced rate/increase/delta with yrate/yincrease/ydelta (and left the latter names available as well).")
```

Comment:
I didn't see any reason to delete the y* names when this setting of "2" is used. In fact I like keeping them, because we can then make sure to have at least one usage by the y* name as a canary that can detect if we're not running our fork (an easy mistake to make).

Reply:
Good call! An easy way to verify we're running what we think we're running!

Reply:
I wonder about making these values a bit more meaningful (but still support the old "1" value for backward compatibility):
```
case "1", "x":
  ...
case "y":
  ...